Gems/LmbrCentral/Code/Source/Shape/EditorReferenceShapeComponent.h

```diff
@@ -24,7 +24,7 @@ namespace LmbrCentral
         static constexpr const char* const s_categoryName = "Shape";
         static constexpr const char* const s_componentName = "Reference Shape";
         static constexpr const char* const s_componentDescription = "Enables the entity to reference and reuse shape entities";
-        static constexpr const char* const s_icon = "Editor/Icons/Components/Vegetation.svg";
+        static constexpr const char* const s_icon = "Icons/Components/Viewport/Component_Placeholder.svg";
```

Comment:
This shouldn't have "Viewport" in the path, it uses a different icon than the Viewport icon.  The regular one is white squares on a transparent background, the viewport one is white squares on a grey background.
![image](https://user-images.githubusercontent.com/82224783/143288207-70bf10fd-3636-4a45-a72c-5a8888bc53d7.png)
![image](https://user-images.githubusercontent.com/82224783/143288250-6f8ca915-31c5-4c97-9f4f-dd195e750aca.png)

Reply:
Changed